contrib/node/examples/src/node/web-build-tool/BUILD

```diff
@@ -6,5 +6,6 @@ node_module(
     'contrib/node/examples/3rdparty/node/null-loader',
     'contrib/node/examples/3rdparty/node/style-loader',
     'contrib/node/examples/3rdparty/node/webpack',
-  ]
+  ],
+  dev_dependency=True,
```

Comment:
Please update the PR description for the new flags, as it will be the commit message.